pkg/network/ebpf/c/protocols/http2/helpers.h

```diff
@@ -35,6 +35,38 @@ static __always_inline bool read_http2_frame_header(const char *buf, size_t buf_
     return true;
 }
 
+// This function reads the http2 frame header and validate the frame.
+static __always_inline bool read_http2_frame_header2(const char *buf, size_t buf_size, struct http2_frame *out) {
```

Comment:
1. let's find a better name
2. why do we have a duplication of the function?

Reply:
Basically the difference  between the functions was that we verified that the type of the current frame is bigger then 9, so I added the check to the "old" read_http2_frame_header and removed the new one. the face that that the type should be bigger then 9 is true always